packages/plugin-session-replay/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
```

Comment:
we can also delete this file and let the publish github action create it from statch